transports/transport-rest/transport-rest-vertx/src/main/java/org/apache/servicecomb/transport/rest/vertx/RestServerVerticle.java

```diff
@@ -117,6 +123,23 @@ public void start(Future<Void> startFuture) throws Exception {
     }
   }
 
+  private void mountGlobalRestFailureHandler(Router mainRouter) {
+    GlobalRestFailureHandler globalRestFailureHandler =
+        SPIServiceUtils.getPriorityHighestService(GlobalRestFailureHandler.class);
+    Handler<RoutingContext> failureHandler = null == globalRestFailureHandler ?
+        ctx -> {
+          LOGGER.error("unexpected failure happened", ctx.failure());
+          ctx.response().setStatusCode(500).putHeader(HttpHeaders.CONTENT_TYPE, MediaType.TEXT_PLAIN)
```

Comment:
It's better all exception thrown by servicecomb using json format and data is CommonExceptionData.

Reply:
fixed
now response body is `{"message":"unknown error"}`, content-type is `application/json`